frontend/src/components/LandingPage/Team/Team.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import './Team.css'
+import {
+    Card, CardHeader, CardTitle, CardText, CardDeck, CardBody, CardColumns
+} from 'reactstrap';
+
+//Displays the team's information using cards. The cards are contained in a Card deck and arranged by card columns.
+// Cards have header and text components with slightly varying colors. Other styling handled by imported CSS file.
+//The entire functional component is contained in a div tag that uses a container to add whitespace.
+//Before the card deck is a header.
+function TeamCardDeck (props) {
```

Comment:
Its usually the convention to name the function the same as the file. What do you think about changing it?

Reply:
Thanks for the feedback Miguel! Implemented the suggested changes.